### PR TITLE
writing_a_great_proposal.md: Line not showing up on website

### PR DESCRIPTION
**Pull request by dushyantRathore**
Last line of writing_great_proposal.md showing up properly on the website with the link.

Closes https://github.com/coala/projects/issues/234

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by hemangsk** [comment]
Can you rectify the shortlog, the commit short log tells about the action performed in that particular commit, ```Line not showing up on website``` is not the description of this commit but it is the issue which this commit solves.

File: data/faq/writing_a_great_proposal.md [hunk 1]
```diff
@@ -22,4 +22,4 @@ Please follow the **application template** in the faq. All fields are required i
 
 For writing your proposal we recommend using [Google Docs](https://www.google.com/docs/about/) as this will be supported by the Google submission system.
 
-If needed, you can get further information about GSoC at <http://write.flossmanuals.net/gsocstudentguide/what-is-google-summer-of-code/>.
\ No newline at end of file
+If needed, you can get further information about GSoC at [http://write.flossmanuals.net/gsocstudentguide/what-is-google-summer-of-code/](http://write.flossmanuals.net/gsocstudentguide/what-is-google-summer-of-code/).
```
Comment (Udayan12167) [inline data/faq/writing_a_great_proposal.md]: You can modify the linke text to here and remove the at. So it would look like `GSoC [here]`.

**Comment by Udayan12167** [comment]
Please rectify review comments by all reviewers :).

**Comment by dushyantRathore** [comment]
Done

**Comment by adtac** [comment]
ack https://github.com/coala/projects/pull/241/commits/5c5a3836fa6f10145088fdc31b9c90879fed7547

**Comment by adtac** [comment]
@dushyantRathore requires rebase

**Comment by dushyantRathore** [comment]
@adtac do I have to rebase?

**Comment by adtac** [comment]
>requires rebase

So yes ;)

**Comment by dushyantRathore** [comment]
Done

**Comment by jayvdb** [comment]
reack d70a832

**Pull request merged**